projects/addon-editor/extensions/file-link/file-link.ts

```diff
@@ -0,0 +1,43 @@
+import {TuiEditorAttachedFile} from '@taiga-ui/addon-editor/interfaces';
+import {Extension} from '@tiptap/core';
+
+declare module '@tiptap/core' {
+    interface Commands<ReturnType> {
+        fileLink: {
+            /**
+             * Comments will be added to the autocomplete.
+             */
+            setFileLink: (preview: TuiEditorAttachedFile) => ReturnType;
+        };
+    }
+}
+
+export const TuiFileLink = Extension.create({
+    name: `fileLink`,
+
+    addCommands() {
```

Comment:
Are we sure want to bundle this particular code? Is it customizable?

Reply:
Yes, you can override

```ts
import(`@taiga-ui/addon-editor/extensions/file-link`).then(
   ({TuiFileLink}) => TuiFileLink.extends({
      addCommands() {
         // custom commands
      }
   }),
)
```